src/shared/js/components/Sidebar/Sidebar.js

```diff
@@ -41,7 +41,7 @@ function TreeNode({ id, field, type, displayText, uuid, childNodes, isExpanded,
   };
 
   return (
-    <li className={`csk-sidebar-node ${isExpanded(uuid) ? 'expanded' : 'collapsed'}`}>
+    <li className={`csk-sidebar-node ${isExpanded(uuid) ? 'expanded' : 'collapsed'}`}  onMouseEnter={handleMouseEnter}>
```

Comment:
Why was this moved back to this element? We moved it to the view link because Adam thought it was too disruptive to have this on the top element.

Reply:
Oh, I was not aware. The UX was weird for me when hovering the list item the page would scroll to the item but I didn't get any highlight.

Reply:
Also... it is a bit weird having a button `view` that on hover does the outline but on click behaves like the edit one.

Reply:
Maybe check with Adam on what he thinks

Reply:
Wait.. it should not do anything on click....